### PR TITLE
DP-21816 Utlity panel title

### PR DESCRIPTION
**Pull request by ygannett**
Any PRs being created needs a changelog.txt file before being merged into dev. See: [Change Log Instructions](https://github.com/massgov/mayflower/blob/develop/docs/for-developers/changelog-instructions.md)

## Related Issue / Ticket

- [JIRA issue](https://jira.mass.gov/browse/DP-21816)
- [Github issue]()

## Steps to Test

1. Go to any page with desktop display.
2. Open the "Log in to...".
3. Inspect the "Log In to..." title in the window.  It's `div`, not `h2` and it maintains the original style.
utility-panel
4. Inspect the emergency alerts header.  It's `div`, not `h2` and it maintains the original style.
alert

5. Build the site in Drupal with this branch, and find the same change as above.
drupal

## Screenshots
Use something like [licecap](http://www.cockos.com/licecap/) to capture gifs to demonstrate behaviors.

## Additional Notes:

Anything else to add?

#### Impacted Areas in Application

*

#### @TODO

*

### CODE REVIEW

File: packages/assets/scss/03-organisms/_utility-nav.scss [hunk 1]
```diff
@@ -377,7 +377,7 @@ $utility-nav-height: 43px;
 
   &__content-title {
 
-    h2 {
+    h2, div {
```
Comment (clairesunstudio) [inline packages/assets/scss/03-organisms/_utility-nav.scss:380]:
```suggestion
    div {

```

**Comment by clairesunstudio** [comment]
@ygannett Are we not changing the `arial-label` for nav elements to match Drupal (#1401)?

**Comment by ygannett** [comment]
@clairesunstudio As you mentioned "Ignore my comments on the nav labeling for now. Talked to Joseph Galluccio this morning and we can address that part later," I kept the `aria-label`s as is for now.  No `aria-label` changes in  #1401 are applied.  Once the labeling is settled, we can update it in both, Mayflower and Drupal, together.

File: packages/assets/scss/03-organisms/_utility-nav.scss (hunk 1 above)
Comment (ygannett) [reply packages/assets/scss/03-organisms/_utility-nav.scss:380]: The h2 is for the React component.

Comment (clairesunstudio) [reply packages/assets/scss/03-organisms/_utility-nav.scss:380]: ah got it ok. Maybe we can do a follow up change in react. thx!

**Comment by clairesunstudio** [comment]
I see, sorry for the confustion. I was referring to this comment: https://github.com/massgov/mayflower/pull/1401#discussion_r622403252

But this looks good to me, we will probably do a major redesign of this component anyways, so keeping the changes simple makes sense to me.

**Review by clairesunstudio — APPROVED** [review]

**Pull request merged**